platform/src/main/java/org/hillview/sketches/MinKRows.java

```diff
@@ -0,0 +1,69 @@
+package org.hillview.sketches;
+
+import it.unimi.dsi.fastutil.longs.Long2IntRBTreeMap;
+import it.unimi.dsi.fastutil.longs.Long2ObjectRBTreeMap;
+import org.hillview.table.api.IColumn;
+
+/**
+ * A data structure that computes the minHash of table. It stores the row indices of the k rows that
+ * hash to the minimum value (for some column to which the hash is applied). We assume no collisions
+ * in the hash function.Finally, we will replace the row indices by the values of the column at
+ * those indices. This structure is used to iterate over the table (and avoid boxing/unboxing).
+ */
+public class MinKRows {
```

Comment:
Maybe we can make this class generic in the type of obejcts it returns?
`MinKRows<T>`